### PR TITLE
Add pipeline to build LoL and Lutris packages at the same time

### PR DESCRIPTION
**Pull request by mandruis7**
This process does It's uses a Ubuntu 18.04 image to 

- Launch and use the LXC cointainers (32bits and 64bits) 

- With the wine version specified in the VERSION file with this parse `$(cat VERSION | cut -d ' ' -f 3 | cut -d '-' -f 1)` 

- Builds the whole wine archive return it to the host machine and release it for both the LoL and Lutris version

For the moment I've it as a manual step but the pipeline could potentially be triggered automatically in each pushed commit

### CODE REVIEW

**Comment by Ahmed-M-Osman1** [comment]
does this fix 5 mints dealy script ?

**Comment by saffanalvy** [comment]
Hi. I'm having a new issue here. I installed Garuda dragonized linux in my brother's laptop then installed lutris and followed all the instructions to install LOL in his laptop just like I installed in my pc. In my pc I could play LOL but in the fresh install in the laptop whenever I run Lol it never prompts the "until I reboot" menu and never asks root password. Thus when I run the game it never works. However, in my pc I used to play Lol but yesterday my Latte dock crashed and I had to reset the config of Latte to default from Garuda assistant and now in my desktop it is not even prompting for that dialog and asking my password and I also can't play this game now. What should I do and how do I fix it? I see there's no issue section enabled so I'm writing here.

**Comment by jarrard** [comment]
> 

You should go to the GE Discord server and ask in wine channel perhaps...

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+  build-wine-ge:
+    runs-on: ubuntu-18.04
+    steps:
+
+      - name: Get branch names
+        id: branch-name
+        uses: tj-actions/branch-names@v5.1
+    
+      - name: Clone repo
+        run: git clone -b ${{ steps.branch-name.outputs.current_branch }} ${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY} . && git submodule update --init buildbot
```
Comment (theoware) [inline .github/workflows/release.yml:14]: Use [actions/checkout](https://github.com/marketplace/actions/checkout)

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+        run: echo ${{ env.RELEASE_VERSION }}
+
+      - name: Install dependencies
+        working-directory: ./buildbot
+        run: |
+          sudo apt-get install -y lxd lxd-client sshpass
+          sudo adduser $USER lxd
+          newgrp lxd
+          id
+
```
Comment (theoware) [inline .github/workflows/release.yml:29]: Remove tabs

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+          sudo adduser $USER lxd
+          newgrp lxd
+          id
+          
+      - name: Setup lxc containers
+        run: |
+          cat buildbot/preseed | sudo lxd init --preseed
+          sudo lxc launch images:ubuntu/bionic/i386 buildbot-bionic-i386
+          sudo lxc launch images:ubuntu/bionic/amd64 buildbot-bionic-amd64
+
```
Comment (theoware) [inline .github/workflows/release.yml:35]: Here too

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+        run: |
+          cat buildbot/preseed | sudo lxd init --preseed
+          sudo lxc launch images:ubuntu/bionic/i386 buildbot-bionic-i386
+          sudo lxc launch images:ubuntu/bionic/amd64 buildbot-bionic-amd64
+      
+      - name: Add and execute 0-buildbot-usersetup.sh
+        run: |
+          sudo lxc exec buildbot-bionic-amd64 -- bash -c "echo -e 'ubuntu\nubuntu' | passwd ubuntu && echo 'ubuntu ALL=NOPASSWD: ALL' | EDITOR='tee -a' visudo"
+          sudo lxc exec buildbot-bionic-i386 -- bash -c "echo -e 'ubuntu\nubuntu' | passwd ubuntu && echo 'ubuntu ALL=NOPASSWD: ALL' | EDITOR='tee -a' visudo"
+
```
Comment (theoware) [inline .github/workflows/release.yml:40]: Also here

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -0,0 +1,303 @@
...
+          scp build.sh ubuntu@buildbot-bionic-amd64:~/buildbot/runners/wine/
+
+      - name: Execute buildbot32-sshsetup
+        run: |
+          
+          ssh ubuntu@buildbot-bionic-i386 'ls ~/.ssh'
+          ssh ubuntu@buildbot-bionic-i386 'sudo chown -R ubuntu:ubuntu ~/.ssh/config'
+          ssh ubuntu@buildbot-bionic-i386 "ssh-keygen -q -t rsa -N '' -f ~/.ssh/id_rsa <<<y >/dev/null 2>&1"
+          ssh ubuntu@buildbot-bionic-i386 'sshpass -p "ubuntu" ssh-copy-id -o StrictHostKeyChecking=no ubuntu@buildbot64'
+
```
Comment (theoware) [inline .github/workflows/release.yml:118]: Here too

File: .github/workflows/release.yml [hunk 6]
```diff
@@ -0,0 +1,303 @@
...
+        run: |
+          ssh ubuntu@buildbot-bionic-i386 "cd buildbot/runners/wine/ && bash patches/protonprep.sh"
+          ssh ubuntu@buildbot-bionic-amd64 "cd buildbot/runners/wine/ && bash patches/protonprep.sh"
+
+      - name: Build Wine GE
+        run: |
+          ssh ubuntu@buildbot-bionic-i386 "cd buildbot/runners/wine && mv wine wine-src"
+          ssh ubuntu@buildbot-bionic-amd64 "cd buildbot/runners/wine && mv wine wine-src"
+          ssh ubuntu@buildbot-bionic-amd64 "cd buildbot/runners/wine && ./build.sh --as lutris --version ${{ env.RELEASE_VERSION }} --usemingw --keep"
+
```
Comment (theoware) [inline .github/workflows/release.yml:136]: Here too

File: .github/workflows/release.yml [hunk 7]
```diff
@@ -0,0 +1,303 @@
...
+        run: echo ${{ env.RELEASE_VERSION }}
+
+      - name: Install dependencies
+        working-directory: ./buildbot
+        run: |
+          sudo apt-get install -y lxd lxd-client sshpass
+          sudo adduser $USER lxd
+          newgrp lxd
+          id
+
```
Comment (theoware) [inline .github/workflows/release.yml:179]: Here aswell

File: .github/workflows/release.yml [hunk 8]
```diff
@@ -0,0 +1,303 @@
...
+          sudo adduser $USER lxd
+          newgrp lxd
+          id
+          
+      - name: Setup lxc containers
+        run: |
+          cat buildbot/preseed | sudo lxd init --preseed
+          sudo lxc launch images:ubuntu/bionic/i386 buildbot-bionic-i386
+          sudo lxc launch images:ubuntu/bionic/amd64 buildbot-bionic-amd64
+
```
Comment (theoware) [inline .github/workflows/release.yml:185]: Here aswell

**Pull request closed without merging**